taxi/admin.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Driver, Manufacturer, Car
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_display = ["model", ]
+    list_filter = ["manufacturer", ]
+    search_fields = ["model", ]
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display =UserAdmin.list_display +("license_number", )
```

Comment:
There is a missing space after the `=` in `list_display =UserAdmin.list_display +("license_number", )`. It should be `list_display = UserAdmin.list_display + ("license_number", )` to maintain consistency with Python's PEP 8 style guide.